lib/galaxy/webapps/galaxy/controllers/authnz.py

```diff
@@ -35,7 +35,7 @@ def index(self, trans, **kwargs):
         rtv = []
         for authnz in trans.user.social_auth:
             rtv.append({'id': trans.app.security.encode_id(authnz.id), 'provider': authnz.provider})
-        return rtv
+        return json.dumps(rtv)
```

Comment:
Instead of this approach, can you try using the @web.json decorator?

Reply:
Thanks, I agree that is cleaner.